### DISCUSSION TITLE
How to install multiple sensors for sniffing in an offline standalone install

### DISCUSSION THREAD

**Discussion by bamaboy5692** [Security-Onion-Solutions/securityonion#3029] [Unsupported Versions]
How can I add multiple sensors for sniffing in a security onion 2.3 standalone install version? Currently only able to see the management interface network

**Comment by weslambert** [comment]
During setup, you should be presented with the various interfaces you can choose to assign to the promiscuous bond.  Is this not the case?

**Reply by bamaboy5692** [reply]
Oh okay. The ENS I remember seeing and it had two interfaces. I think maybe it's just a configuration issue then. I am going to try going into the configuration and changing the IP. Currently it's only seeing my management laptop on the dashboards.
